packages/dso-toolkit/src/components/card/card.content.ts

```diff
@@ -43,3 +43,14 @@ export const cardContentLabel: Card<unknown> = {
     },
   ],
 };
+
+export const cardContentSlideToggle: Card<unknown> = {
+  ...cardContent,
+  interactions: [
+    {
+      dsoActiveChange: () => {},
```

Comment:
Het is fraaier om `dsoActiveChange` in `card.models.ts` een optionele property van te maken ipv. een lege listener. Voor listeners heeft dit altijd de voorkeur. We hoeven immers niet per se te "luisteren" naar events in een component. Dit is slordigheid dat we dit niet meteen zo hebben gedaan.

Reply:
dsoActiveChange zit in `slide-toggle.models.ts` en aldaar heb ik die optioneel gemaakt